webpack.config.js

```diff
@@ -16,13 +16,15 @@ const CssMinimizerPlugin = require("css-minimizer-webpack-plugin");
 const TerserPlugin = require("terser-webpack-plugin");
 const dedent = require("dedent");
 const { Compilation } = require("webpack");
+const { exec } = require("child_process");
 
 /*******************************************************************************
  * Paths for various assets (sources and destinations)
  */
 
 const scriptPath = resolve(__dirname, "src/pydata_sphinx_theme/assets/scripts");
 const staticPath = resolve(__dirname, "src/pydata_sphinx_theme/theme/pydata_sphinx_theme/static");
+const localePath = resolve(__dirname, "src/pydata_sphinx_theme/locale");
```

Comment:
I do not understand why this path wasn't always `resolve(staticPath, "/locale")`

Reply:
I do not know

Reply:
My comment wasn't clear. There is a mismatch between the locale path here and the locale path specified in pyproject.toml. It specifies `"locale/"` within the `additional-compiled-static-assets` property for sphinx-theme-builder. If you work backwards from the line [`self.theme_static_path / asset_name`](https://github.com/pradyunsg/sphinx-theme-builder/blob/45d3f2f53f190864f91f8aa98f52ad8bab178f4e/src/sphinx_theme_builder/_internal/project.py#L391C22-L391C39) in the sphinx-theme-builder source code, then you will see that the path part specified in pyproject.toml ultimately resolves to `src/pydata_sphinx_theme/theme/pydata_sphinx_theme/static/locale`.

These are completely different locations in the source code directory, so I'm wondering why there is this discrepancy.

Reply:
Well yes, files in `src/pydata_sphinx_theme/assets` are considered static assets that need to be compiled (source `scss, JS`, etc.) while `src/pydata_sphinx_theme/theme/pydata_sphinx_theme/static/` are the compiled assets (see for example the structure that stb expects https://sphinx-theme-builder.readthedocs.io/en/latest/filesystem-layout/)

It seems that we are following a similar pattern here where `src/pydata_sphinx_theme/locale` are the source (`.po, .pot` ) catalogue files and the compiled `.mo` files are in `src/pydata_sphinx_theme/theme/pydata_sphinx_theme/static/locale` (which is what confused me a bit).
Other themes seem to have locales in `src/theme/locale` (both source and compiled) so we could change that to align with other themes but for now I think we should just get the expected / default behahviour working again.